pypdf/filters.py

```diff
@@ -481,6 +486,52 @@ def decode(
         return data
 
 
+class BrotliDecode:
+    """Decodes Brotli-compressed data (ISO 32000-2:2020, Section 7.4.11)."""
+    @staticmethod
+    def decode(
+        data: bytes,
+        decode_parms: Optional[DictionaryObject] = None,
+        **kwargs: Any,
+    ) -> bytes:
+        """
+        Decode Brotli-compressed data.
+
+        Args:
+            data: Brotli-compressed data.
+            decode_parms: Optional parameters (unused).
+
+        Returns:
+            Decompressed data.
+
+        Raises:
+            ImportError: If brotli library is not installed.
+        """
+        if brotli is None:
+            # Raise ImportError as the required library is missing
+            raise ImportError("Brotli library not installed. Required for BrotliDecode filter.")
+        return brotli.decompress(data)
+
+    @staticmethod
+    def encode(data: bytes, **kwargs: Any) -> bytes:
+        """
+        Encode data using Brotli compression.
+
+        Args:
+            data: Data to compress.
+
+        Returns:
+            Compressed data.
+
+        Raises:
+            ImportError: If brotli library is not installed.
+        """
+        if brotli is None:
+            # Raise ImportError as the required library is missing
+            raise ImportError("Brotli library not installed. Required for BrotliDecode filter.")
```

Comment:
See above for the docstring and the comment. Additionally, a test is missing here.